### ISSUE TITLE
FormControl is breaking Popover

### ISSUE THREAD

**Issue by patelnav** [react-bootstrap/react-bootstrap#3163]
Hey everyone,
Looks like there's some issue where FormControl is conflicting for focus with Popover.
If you replace FormControl with <input className="form-control"/> this bug goes away.

[Here's a JSFiddle](https://jsfiddle.net/patelnav/o9qvrdx5/) showing the bug

**Comment by taion** [comment]
no, that's just because you have the trigger inside a `<label>`; your latter example doesn't show this because the label isn't attached to the input (i.e. you're missing `id="formTestWorks"` on your `<input>`)

**Comment by patelnav** [comment]
Great! Thanks, in case anyone comes upon this via searches here's [a fixed version](https://jsfiddle.net/patelnav/o9qvrdx5/7/)

**Comment by patelnav** [comment]
@taion what is the reason trigger won't work inside a <label>? 
(trying to understand this better)
